app/src/main/res/layout/home_fragment.xml

```diff
@@ -25,28 +32,49 @@
             android:gravity="center"
             android:padding="@dimen/home_fragment_padding"
             android:text="@string/pslab_device_not_found"
+            android:textColor="@color/blue_grey"
             android:textSize="@dimen/device_status_textsize"
             android:textStyle="bold" />
+
+        <TextView
+            android:id="@+id/tv_connect_msg"
+            android:layout_width="match_parent"
+            android:layout_height="@dimen/device_status_height"
+            android:gravity="top|center"
+            android:paddingLeft="@dimen/padding_20dp"
+            android:paddingRight="@dimen/padding_20dp"
+            android:text="@string/please_connect_the_ps_lab_device_through_otg"
+            android:textColor="@color/blue_grey"
+            android:textSize="@dimen/device_status_textsize"
+            android:textStyle="bold" />
+
+        <TextView
+            android:id="@+id/tv_device_version"
+            android:layout_width="match_parent"
+            android:layout_height="@dimen/device_status_height"
+            android:gravity="top|center"
+            android:paddingLeft="@dimen/padding_20dp"
+            android:paddingRight="@dimen/padding_20dp"
+            android:text="@string/temp_text"
+            android:textColor="@color/blue_grey"
+            android:textSize="@dimen/device_status_textsize"
+            android:textStyle="bold"
+            android:visibility="gone" />
     </LinearLayout>
 
-    <TextView
-        android:id="@+id/tv_device_version"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:layout_marginTop="@dimen/home_fragment_textview_margin"
-        android:gravity="center"
-        android:padding="@dimen/home_fragment_padding"
-        android:textSize="@dimen/device_status_textsize"
-        android:textStyle="bold" />
 
     <TextView
-        android:id="@+id/tv_initialisation_status"
-        android:layout_width="match_parent"
+        android:id="@+id/tv_device_description"
+        android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_marginTop="@dimen/home_fragment_textview_margin"
-        android:gravity="center"
+        android:layout_gravity="center"
+        android:layout_marginTop="30dp"
         android:padding="@dimen/home_fragment_padding"
+        android:text="@string/what_is_ps_lab_device"
+        android:textAppearance="@style/Base.Widget.AppCompat.Spinner.Underlined"
+        android:textColor="#C72C2C"
```

Comment:
You can get this color from the resource file. I see that you have done it in other places but not here.